### ISSUE TITLE
Bug report about gameplay

### ISSUE THREAD

**Issue by evorios** [EaW1805/www#16]
[Continuing the theme](http://forum.eaw1805.com/viewtopic.php?f=48&t=924):

1) Game 30: mechanics of fixed alliance works with faults

> So someone declared war to you, or your ally, and you/he did not receive a message asking to join the war?

Yes. This happens all the time since I broke the alliance with Egypt. In [the previous step](http://www.eaw1805.com/report/scenario/1805/game/30/nation/15/month/24), I declared war against Poland, but Egypt did not react to it - he did not get a warning (established relationships - "Trade").

2) Tactical battle works with faults

In [this battle](http://www.eaw1805.com/report/scenario/1805/game/30/nation/15/battle/3755) I fought against enemy alone. Although [previous fight](http://www.eaw1805.com/report/scenario/1805/game/30/nation/15/battle/3743) was with the ally.

3) I guarded the transition to the other side of the Strait, but the enemy walked past it not paying attention to the presence of a fortified fort on its shore.
![2015-09-28_153448-min](https://cloud.githubusercontent.com/assets/1260935/10134440/b99bedba-65f6-11e5-90e2-8702f509e701.png)

**Comment by Silver1974** [comment]
Dear Roman,

are you by any chance in enforced peace with any of the parties that you
wish to be called into war? Because if that is the case, you do not get the
"call to ally" message since you cannot declare war.

Regarding the battles, I have been asked the same by your opposition. It is
a rare case of 2 vs 2 where one from each side is _not_ at war with one
from the other side (due to enforced peace). In this case, you cannot fight
2 vs 2, because you are not all enemies & allies. So it will be 2 to 1,
with the defender (owner of the sector) always participating, but the rest
being randomly rolled.

For everything else we will look at it.
Makis

**Comment by evorios** [comment]
> are you by any chance in enforced peace with any of the parties that you wish to be called into war?

Nope. I only send letters with peace offering to leaders. But not order it.

**Comment by evorios** [comment]
My defective alliance continues to "rejoice" by new bugs:
![2015-10-03_130332](https://cloud.githubusercontent.com/assets/1260935/10262234/9fd1537a-69cf-11e5-9cb4-f6691c620980.png)
![2015-10-03_130431](https://cloud.githubusercontent.com/assets/1260935/10262233/9fcd865a-69cf-11e5-8501-ed99dcf0b3dc.png)

**Comment by evorios** [comment]
It is bug?
![2015-10-03_132311](https://cloud.githubusercontent.com/assets/1260935/10262272/3b293ab6-69d2-11e5-863f-a8b8bdc2055f.png)
![2015-10-03_132445](https://cloud.githubusercontent.com/assets/1260935/10262271/3b271b0a-69d2-11e5-8873-087019154a87.png)
![2015-10-03_132746](https://cloud.githubusercontent.com/assets/1260935/10262277/919d9d2e-69d2-11e5-98e6-9b4bcc8b7ca8.png)
